### PR TITLE
[doc] Update editorial workflow content and images 

### PR DESCRIPTION
**Pull request by SnowMoon-Dev**
## Brief change log

1. update editorial workflow content and images
2. update project overview picture

## Verify this pull request

Manually verified the change by testing locally

### CODE REVIEW

File: docs/docs/en/guide/project/workflow-instance.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Click Project Management -> Workflow -> Workflow Instance, enter the workflow in
 
 ![workflow-instance](/img/new_ui/dev/project/workflow-instance.png)
 
-- **Edit：** only can edit terminated processes. Click the "Edit" button or the workflow instance name to enter the DAG edit page. After the edit, click the "Save" button to confirm, as shown in the figure below. In the pop-up box, check "Whether to update to workflow definition" to update the workflow definition; if not checked, no effect on the workflow definition
+- **Edit：** only can edit terminated processes. Click the "Edit" button or the workflow instance name to enter the DAG edit page. After the edit, click the "Save" button to confirm, as shown in the figure below. In the pop-up box, check "Whether to upload to workflow definition" to update instance modified information to workflow definition; if not checked, no effect on the workflow definition
```
Comment (SbloodyS) [inline docs/docs/en/guide/project/workflow-instance.md]: Only workflow instance state in ```successed/failed/killed``` can be edited.

File: docs/docs/zh/guide/project/workflow-instance.md [hunk 2]
```diff
@@ -40,7 +40,7 @@
 
 ![workflow-instance](/img/new_ui/dev/project/workflow-instance.png)
 
-- **编辑：** 只能编辑已终止的流程。点击"编辑"按钮或工作流实例名称进入 DAG 编辑页面，编辑后点击"保存"按钮，弹出保存 DAG 弹框，如下图所示，在弹框中勾选"是否更新到工作流定义"，保存后则更新工作流定义；若不勾选，则不更新工作流定义。
+- **编辑：** 只能编辑已停止的流程。点击"编辑"按钮或工作流实例名称进入 DAG 编辑页面，编辑后点击"保存"按钮，弹出保存 DAG 弹框，如下图所示，修改流程定义信息，在弹框中勾选"是否上传流程定义"，保存后则将实例修改的信息更新到工作流定义；若不勾选，则不更新工作流定义。
```
Comment (SbloodyS) [inline docs/docs/zh/guide/project/workflow-instance.md]: 只有工作流实例状态在```成功/失败/停止```中的时候可以被编辑。

Comment (SnowMoon-Dev) [reply docs/docs/zh/guide/project/workflow-instance.md]: 好的，我更改了相应的内容

File: docs/docs/en/guide/project/workflow-instance.md (hunk 1 above)
Comment (SnowMoon-Dev) [reply docs/docs/en/guide/project/workflow-instance.md]: ok, I modified the corresponding content

File: docs/docs/zh/guide/project/workflow-instance.md [hunk 3]
```diff
@@ -40,7 +40,7 @@
 
 ![workflow-instance](/img/new_ui/dev/project/workflow-instance.png)
 
-- **编辑：** 只能编辑已终止的流程。点击"编辑"按钮或工作流实例名称进入 DAG 编辑页面，编辑后点击"保存"按钮，弹出保存 DAG 弹框，如下图所示，在弹框中勾选"是否更新到工作流定义"，保存后则更新工作流定义；若不勾选，则不更新工作流定义。
+- **编辑：** 只能编辑 成功/失败/停止 状态的流程。点击"编辑"按钮或工作流实例名称进入 DAG 编辑页面，编辑后点击"保存"按钮，弹出保存 DAG 弹框，如下图所示，修改流程定义信息，在弹框中勾选"是否上传流程定义"，保存后则将实例修改的信息更新到工作流定义；若不勾选，则不更新工作流定义。
```
Comment (SbloodyS) [inline docs/docs/zh/guide/project/workflow-instance.md]: 是否上传流程定义 修改为 是否更新工作流定义

File: docs/docs/en/guide/project/workflow-instance.md [hunk 4]
```diff
@@ -40,7 +40,7 @@ Click Project Management -> Workflow -> Workflow Instance, enter the workflow in
 
 ![workflow-instance](/img/new_ui/dev/project/workflow-instance.png)
 
-- **Edit：** only can edit terminated processes. Click the "Edit" button or the workflow instance name to enter the DAG edit page. After the edit, click the "Save" button to confirm, as shown in the figure below. In the pop-up box, check "Whether to update to workflow definition" to update the workflow definition; if not checked, no effect on the workflow definition
+- **Edit：** only processes with success/failed/stop status can be edited. Click the "Edit" button or the workflow instance name to enter the DAG edit page. After the edit, click the "Save" button to confirm, as shown in the figure below. In the pop-up box, check "Whether to upload to workflow definition" to update instance modified information to workflow definition; if not checked, no effect on the workflow definition
```
Comment (SbloodyS) [inline docs/docs/en/guide/project/workflow-instance.md]: ```Whether to upload to workflow definition``` change to ```Whether to update to workflow definition```

```no effect on the workflow definition``` change to ```the workflow definition would not be updated```.

Comment (SnowMoon-Dev) [reply docs/docs/en/guide/project/workflow-instance.md]: @SbloodyS Thanks, I have modified the relevant content.

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by Amy0104 — APPROVED** [review]
LGTM.

**Comment by SbloodyS** [comment]
@zhongjiajie Do you have any other better suggestions?

**Comment by zhongjiajie** [comment]
> @zhongjiajie Do you have any other better suggestions?

no, well done @SnowMoon-Dev  and thanks for you contribution, merging

**Pull request merged**